test/daypickerinput/rendering.js

```diff
@@ -174,5 +174,21 @@ describe('DayPickerInput', () => {
       expect(spy).toHaveBeenCalledTimes(3);
       spy.mockRestore();
     });
+
+    it('should use `selectedDays` from DayPickerInput props', () => {
+      const wrapper = mount(
+        <DayPickerInput
+          selectedDays={new Date(2017, 1, 8)}
+          dayPickerProps={{
+            month: new Date(2017, 1),
+            selectedDays: new Date(2017, 1, 9),
+          }}
+        />
+      );
+      wrapper.instance().showDayPicker();
+      wrapper.update();
+      expect(wrapper.find('.DayPicker-Day--selected')).toHaveLength(1);
+      expect(wrapper.find('.DayPicker-Day--selected').at(0)).toHaveText('8');
```

Comment:
So this test is OK, it should have picked the `selectedDays` from `dayPickerProps`. No need to set `selectedDays` in the `<DayPickerInput />`.

The tests we are missing is: 
* if `selectedDays` is passed to `dayPickerProps`, clicking a day in the day picker will still use the days from `dayPickerProps`
* if `selectedDays` is passed to `dayPickerProps`, typing a new valid day in the day picker will still use the days from `dayPickerProps`

Reply:
Oh, ok, I'll remove that extra prop, but what to do with behavior that checks by [this](https://github.com/gpbl/react-day-picker/blob/master/test/daypickerinput/events.js#L248) test?